projects/plugins/jetpack/modules/subscriptions/newsletter-widget/src/components/newsletter-widget.tsx

```diff
@@ -108,34 +108,36 @@ export const NewsletterWidget = ( {
 					<h3 className="newsletter-widget__heading">{ __( 'Quick Links', 'jetpack' ) }</h3>
 					<ul className="newsletter-widget__footer-list">
 						<li>
-							<a href={ `${ adminUrl }edit.php` }>{ __( 'Publish your next post', 'jetpack' ) }</a>
+							<a href={ `${ adminUrl }post-new.php` }>
```

Comment:
I also took the opportunity to update the link to publish a new post to point to the post editor, it seemed more appropriate.